### PR TITLE
Set default RLC flags

### PR DESCRIPTION
**Pull request by joheredi**
Setting a few commonly used RLC flags to minimize the boilerplate needed when creating a new config file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,14 @@ output-artifact: typescript-files
 ### REST Client Generator overrides
 
 ```yaml $(rest-level-client)
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+source-code-folder-path: ./src
```
Comment (qiaozha) [inline README.md:92]: I don't think that's applicable for multi-client ?

File: README.md [hunk 2]
```diff
@@ -87,6 +87,14 @@ output-artifact: typescript-files
 ### REST Client Generator overrides
 
 ```yaml $(rest-level-client)
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+source-code-folder-path: ./src
+add-credentials: true
+
+typescript:
+  output-folder: ../
```
Comment (qiaozha) [inline README.md:96]: Same here

File: README.md (hunk 1 above)
Comment (joheredi) [reply README.md:92]: When adding multi client support we would probably need to add another check for the multi-client flag and setup the right directives for this.  Such as `$(rest-level-client) && !$(multi-client)`

Comment (dw511214992) [reply README.md:92]: For multi-client, we can define the `source-code-folder-path` in the README.md which is stored in sdk repo. Then the `source-code-folder-path` will be overrided.

File: README.md [hunk 3]
```diff
@@ -117,12 +116,23 @@ pipeline:
...
     scope: typescript-scope/emitter
-
 ```
 
 ### REST Client Generator overrides
 
 ```yaml $(rest-level-client)
+license-header: MICROSOFT_MIT_NO_VERSION
+
+$(add-credentials == undefined):
```
Comment (qiaozha) [inline README.md:126]: Are you sure this kind of yaml condition really work ?

**Pull request closed without merging**